draft-ietf-tls-esni.md

```diff
@@ -476,6 +473,10 @@ contain a "cached_info" extension {{!RFC7924}} with a CachedObject
 entry whose CachedInformationType is "cert", since this indication
 would divulge the true server name.
 
+Finally, client computes a PSK binder using outer_binder_key with the identity
+outer_binder_identity. This binds the encryption of ClientHelloInner to
+ClientHelloOuter.
```

Comment:
This probably wants an extra sentence that you add the identity and binder to the PSK extension. At first I was confused what you did with the binder.